src/xdocs/writingchecks.xml

```diff
@@ -656,20 +656,6 @@ public class MethodLimitCheck extends AbstractCheck
         There are basically only a few limits for Checkstyle:
       </p>
       <ul>
-        <li>Java
-          <a href="https://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-3.5">
-            tokens (identifiers, keywords)</a>
-          should be written with
-          <a href="https://en.wikipedia.org/wiki/ASCII">ASCII</a>
-          characters ONLY,
-          no
-          <a href="https://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-3.3">
-            Unicode escape</a>
```

Comment:
Does the java compiler support delimited unicode? Ex: `\u0030` .
If so, is that something we now support?

Reply:
> Does the java compiler support delimited unicode?

Yes, it does.

> If so, is that something we now support?

No, we do not. I explored this a bit, and lexer performance was definitely affected. I am not sure if it is worth it to add support for this.

Reply:
Good catch here, I have updated documentation to be very specific.